uvloop/loop.pxd

```diff
@@ -1,15 +1,35 @@
 # cython: language_level=3
 
+cdef extern from *:
+    '''
+    enum { UV_STREAM_RECV_BUF_SIZE = 256000,
+           SSL_READ_MAX_SIZE = 256 * 1024,  // 250kb
```

Comment:
This is already defined as a `compile_time_env`?

Reply:
It may not be needed now as I was trying to get cython3 working without DEF